README.md

```diff
@@ -13,6 +13,11 @@ All components of the LINSTOR software stack can be managed by the operator:
 * LINSTOR CSI driver
 
 
+## Legacy Operator
+
+The previous version of Piraeus Operator is still available [here.](https://github.com/piraeusdatastore/piraeus-operator/tree/master)
+It will be maintained, receiving updates to fix issues or new software versions until a stable upgrade path to V2 is available.
```

Comment:
Nit:

```suggestion
It will be maintained, receiving updates to fix issues or new software versions until a stable upgrade path to v2 is available.
```

It is written this way in the paragraph below.